spec/linter-codeclimate-spec.js

```diff
@@ -35,4 +37,21 @@ describe('The codeclimate provider for Linter', () => {
       ),
     ),
   );
+
+  it('runs all configured engines', () =>
```

Comment:
Since this PR is removing all pre-defined mappings, is this spec actually doing anything different from the one on L22?